okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -609,53 +608,55 @@ private void proxyRequestHttpClient(
     RoutingContext ctx = pc.getCtx();
     String url = makeUrl(mi, ctx);
     HttpMethod meth = ctx.request().method();
-    HttpClientRequest clientRequest = httpClient.requestAbs(meth, url, res1 -> {
-      if (proxyHttpFail(pc, mi, res1)) {
```

Comment:
this refactoring is annoying, but at least this is cleaner...

Reply:
Yes.. It is a better API..